### PR TITLE
Add account_emergency_expire_date to OL7 stig

### PR DESCRIPTION
**Pull request by freddieRv**
#### Description:

- Add `account_emergency_expire_date` rule to the OL7 stig profile

#### Rationale:

- OL7 DISA STIG efforts

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_emergency_expire_date/rule.yml
Comment (Mab879) [inline linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_emergency_expire_date/rule.yml:46]: Please ensure that these keys are sorted.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR; just one minor change is needed.

File: linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_emergency_expire_date/rule.yml
Comment (freddieRv) [reply linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_emergency_expire_date/rule.yml:45]: What a dumb mistake sorry. Addressed 👍🏻

**Review by Mab879 — APPROVED** [review]
Thanks for the PR! LGTM. Overriding the CODEOWNERS file since @freddieRv can't approve this PR.

**Comment by Mab879** [comment]
Waving Ansible lint as these changes did not change any Ansible remediations.

**Pull request merged**
